libs/barista-components/schematics/src/migrations/update-7-0-0/config/template-replacements.ts

```diff
@@ -0,0 +1,201 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TemplateReplacementList } from '../rules';
+import { replaceHtmlAttribute } from '../utils/replace-html-attribute';
+
+export const TEMPLATE_REPLACEMENTS: TemplateReplacementList[] = [
+  {
+    needle: 'aria-label-close',
+    replacement: (content) =>
+      replaceHtmlAttribute(
+        'aria-label-close',
+        'ariaLabelClose',
+        // TODO: check the list of all elements if nothing is missing
```

Comment:
Looks good, only components i found are range and timestamp. The close button is on the context-dialog which is handled below